### PR TITLE
first refactoring of README.md

### PR DESCRIPTION
**Pull request by stefanscheidt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,69 @@
...
+
+## Kata Description
+
+Origin: [Train Reservation Kata](https://github.com/emilybache/KataTrainReservation) from Emily Bache
 
 Railway operators aren't always known for their use of cutting edge technology, and in this case they're a little behind the times. The railway people want you to help them to improve their online booking service. They'd like to be able to not only sell tickets online, but to decide exactly which seats should be reserved, at the time of booking.
 
-You're working on the "TicketOffice" service, and your next task is to implement the feature for reserving seats on a particular train. The railway operator has a service-oriented architecture, and both the interface you'll need to fulfill, and some services you'll need to use are already implemented.
+You're working on the "Ticket Office Service", and your next task is to implement the feature for reserving seats on a particular train. The railway operator has a service-oriented architecture, and both the interface you'll need to fulfill, and two services you'll need to use - "Booking Reservation Service" and "Train Data Service" - are already implemented.
```
Comment (mrclneumann) [inline README.md:11]: Booking Reference Service instead of Booking Reservation Service, right?

File: README.md [hunk 2]
```diff
@@ -1,77 +1,69 @@
...
-## Business Rules around Reservations
+### Business Rules around Reservations
 
 There are various business rules and policies around which seats may be reserved. For a train overall, no more than 70% of seats may be reserved in advance, and ideally no individual coach should have no more than 70% reserved seats either. However, there is another business rule that says you _must_ put all the seats for one reservation in the same coach. This could make you and go over 70% for some coaches, just make sure to keep to 70% for the whole train.
 
-## The Guiding Test
+### The Guiding Test
 
-The Ticket Office service needs to respond to a HTTP POST request that comes with form data telling you which train the customer wants to reserve seats on, and how many they want. It should return a json document detailing the reservation that has been made. 
+The Ticket Office service needs to respond to a HTTP POST request that comes with form data telling you which train the customer wants to reserve seats on, and how many they want. It should return a JSON document detailing the reservation that has been made.
```
Comment (mrclneumann) [inline README.md:19]: We haven't specified this yet but are we expecting form data here from the participants? I assume we should change this to a JSON body similar to the upstream services.

File: README.md (hunk 1 above)
Comment (stefanscheidt) [reply README.md:11]: Fixed.

File: README.md (hunk 2 above)
Comment (stefanscheidt) [reply README.md:19]: I updated the README to reflect that the train reservation service should accept a JSON body.

**Pull request merged**